src/store/redusers/get_data_reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { SET_DATA } from '../constants';
+
+const initialState = {
+  inverter_one: [],
+  inverter_two: [],
+};
+
+export const set_data_reducer = (state = initialState, action: { type: string; value: any; }) => {
```

Comment:
generally speaking, action must only contain type. And it may also contain value, or some other field like 'data' or 'payload' or whatever. Probably action type should be described in some other, more general way. Consider using an interface instead of type.

Reply:
did something, but there are tons of questions